test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -19,7 +19,7 @@ public DaemonConfiguration(string configYamlFile, Option<string> agentImage, Opt
             string contents = File.ReadAllText(this.configYamlFile);
             this.config = new YamlDocument(contents);
             this.UpdateAgentImage(
-                agentImage.GetOrElse("mcr.microsoft.com/microsoft/azureiotedge-agent:1.0"),
```

Comment:
Do we accidentally have an image here, or was this just not working if agent image wasn't supplied?

Reply:
I don't think this ever worked. It is only used in a very specific situation: running the tests manually, without specifying "edgeAgentBootstrapImage" in context.json.